vlsi/example-vlsi

```diff
@@ -24,21 +24,25 @@ def example_tool_settings(x: hammer_vlsi.HammerTool) -> bool:
     x.append('''
 # TODO
 # Place custom TCL here
+set_db route_design_bottom_routing_layer 2
+set_db route_design_top_routing_layer 7
 ''')
     return True
 
 def scale_final_gds(x: hammer_vlsi.HammerTool) -> bool:
     """
     Scale the final GDS by a factor of 4
+    hammer/src/hammer-vlsi/technology/asap7/__init__.py implements scale_gds_script
     """
     x.append('''
 # Write script out to a temporary file and execute it
 set fp [open "{script_file}" "w"]
 puts -nonewline $fp "{script_text}"
 close $fp
-if {{ [catch {{ exec python3 {script_file} }} msg] }} {{
-    puts "$::errorInfo"
-}}
+
+# Innovus <19.1 appends some bad LD_LIBRARY_PATHS, so remove them before executing python
```

Comment:
Should this also include an example of the checking versions API?

Reply:
Yeah, I think I could check for version 181 and below or error out. This particular comment doesn't need that because P&R should not be attempted with version 191 anyway.